parsl/executors/radical/executor.py

```diff
@@ -361,14 +363,25 @@ def unwrap(self, func, args):
 
         return func, args, task_type
 
-    def task_translate(self, tid, func, args, kwargs):
+    def task_translate(self, tid, func, parsl_resource_specification, args, kwargs):
         """
         Convert parsl function to RADICAL-Pilot rp.TaskDescription
         """
 
         task = rp.TaskDescription()
         task.name = func.__name__
         task.timeout = kwargs.get('walltime', 0.0)
+
+        if parsl_resource_specification and isinstance(parsl_resource_specification, dict):
+            logger.debug('mapping parsl resource sepcfications >> rp resource sepcfications')
+            for key, val in parsl_resource_specification.items():
+                if key not in task.as_dict():
+                    key = PARSL_RP_RESOURCE_MAP.get(key, None)
+                    if not key:
+                        logger.warning('ignoring task {0} as it is not supported by RP'.format(key))
```

Comment:
this is ignoring a resource specification key, not a task?

Reply:
This will only ignore the key.

Reply:
maybe log message is better: ignoring key 'memory'...